dashboard/test/models/script_test.rb

```diff
@@ -1150,6 +1150,21 @@ def populate_cache_and_disconnect_db
     assert_equal [script], scripts
   end
 
+  test "get_assessment_script_levels returns an empty list if no level groups" do
+    script = create(:script, name: 'test-no-levels')
+    level_group_script_level = script.get_assessment_script_levels
+    assert_equal level_group_script_level, []
+  end
+
+  test "get_assessment_script_levels returns a list of script levels" do
+    script = create(:script, name: 'test-level-group')
+    level_group = create(:level_group, name: 'assessment 1')
```

Comment:
nit: `name: ` does not look necessary, here and above

Reply:
I thought the same thing, but apparently it's required because I get an error without it. 🤷‍♀️

Reply:
good point :-) apparently this is the way to make a param that is required on the model not be required when calling the factory:
https://github.com/code-dot-org/code-dot-org/blob/a9b35040af512661dd7f1c6181aa70317b5dd05b/dashboard/test/factories/factories.rb#L9-L11